pkg/pillar/cmd/zedrouter/zedrouter.go

```diff
@@ -754,38 +762,6 @@ func updateLispConfiglets(ctx *zedrouterContext, legacyDataPlane bool) {
 // Wrappers around handleCreate, handleModify, and handleDelete
 
 // Determine whether it is an create or modify
-func handleAppNetworkConfigModify(ctxArg interface{}, key string, configArg interface{}) {
-
-	ctx := ctxArg.(*zedrouterContext)
-	config := cast.CastAppNetworkConfig(configArg)
-	log.Infof("handleAppNetworkConfigModify(%s-%s)\n", config.DisplayName, key)
-
-	// If this is the first time, update the timer for GC
```

Comment:
You appear to have lost like 763-767. I assume the first time is always a create (and not a modify) so this needs to go in the create handler.